docs/source/development.rst

```diff
@@ -0,0 +1,104 @@
+
+We do not have a concrete plan about versioning strategy after v1.0.0.
+
+
+Compatibiity policy
+===================
+
+As an immediate consequence of the semantic versioning, we may break compatibility of public APIs including addition, deletion, change in semantics of them anytime in the major version zero.
+Since APIs of Chainer Chemistry are still immature and unstable, introduction of new features can sometime involve compatibility break.
+If we are faced with a dilemma between cost for backward compatibility and benefit of new features, we are likely to give up the former because we want to place importance on introducing new features as early as possible. Of course, we care backward compatibility whenever it is easy and low-cost.
+
+Like `ChainerCV <https://twitter.com>`_, Chainer Chemistry provides several off-the-shelf deep learning models (e.g. Neural Finger Print) whose papers are available in such as arXiv or conferences related to machine learning.
+Although, most of published papers reports evaluation results of the models with publicly available datasets, we do *NOT* guarantee the reproducibility of experiments in the papers.
+
+At some point, coding examples in the master branch of the official repository may not work even with the latest release. In that case, users are recommended to either use the example code of the latest release or update the library code to the master branch.
+
+Branch strategy
+===============
+
+The official repository of Chainer Chemistry is https://github.com/pfnet-research/chainer-chemistry. 
+We use the *master* branch of the repository for development. Therefore, developer who makes PRs should send them to the master branch.
+
+During major version zero, we do not maintain any released versions.
+When a bug is found, changes for the bug should be merged to the next version (either minor or patch). If the bug is critical, we will release the next version as soon as possible.
+
+
+Coding guideline
+================
+
+We use upper camel case (e.g. ``FooBar``) for class names and snake case (e.g. ``foo_bar``) for function, method, variable and package names.
+
+
+Testing guideline
+=================
+
+Chainer Chemistry uses `pytest <https://docs.pytest.org/en/latest/index.html>`_  as a unit-test framework.
+All unit tests are located in ``tests/`` directory. We can run tests with normal usage of pytest.
+For example, the following command runs all unit tests::
+
+   $ pytest tests
+
+Some unit tests require GPUs, which are annotated with ``@pytest.mark.gpu``.
+Therefore, you can skip them with ``-m`` option::
+
+   $ pytest -m "not gpu" tests
+
+If a develop who write a unit test that uses GPUs, you must anotate it with ``@pytest.mark.gpu``.
+
+Similarly, some unit tests take long time to complete.
+We annotated them with ``@pytest.mark.slow`` and skip them with ``-m`` option::
```

Comment:
We annotated them with ``@pytest.mark.slow`` and can be skipped with ``-m`` option::

Reply:
I think this sentence implicitly changes a subject. I'll change as "We annotate them with `@pytest.mark.slow` and skip them with ...".